### DISCUSSION TITLE
Stringtie Input Questions - To Merge or Not to Merge?

### DISCUSSION THREAD

**Discussion by AwesomeTimeTraveler** [kvittingseerup/IsoformSwitchAnalyzeR#233] [Q&A]
I have wanted to use the package IsoformSwitchAnalyzeR to look at the transcriptomic data I have generated in several species of rodents (including non-traditional novel models), and wanted to ask about the inputs.

Our transcriptomic pipeline for bulk RNAseq uses Hisat2-Stringtie-DESeq2, where we merge the counts from each individual sample to make a merged GTF "de novo transcriptome" to count each sample against. This is a slight modification of the pipeline listed here (https://www.nature.com/articles/nprot.2016.095), where we are instead not using Ballgown for differential modeling due to the small N of our groups.

To input this data into IsoformSwitchAnalyzeR, do I need to run Stringtie with -b/-B to create a .ctab file for each individual sample, skipping the merge, and input these directly in a separate sub-directory per sample? 

Or would I miss out on the quality of counts due to the way Stringtie is working internally with the merged count method referenced above? Loading just the merged .ctab makes no sense, so I assume we would want the individual .ctab files but wanted to confirm.

**TLDR** - Do you input individual Ballgown format .ctab files for each sample, with no merging, or some other style?

If relevant, I can provide the exact flags used in all steps of the above pipeline for our particular data.

Thanks!

**Comment by AwesomeTimeTraveler** [comment]
The vignette did not seem to clarify. I am quite new to bioinformatics, so hopefully someone can guide me!
